src/components/Button/button.module.scss

```diff
@@ -163,6 +163,13 @@
         width: fit-content;
     }
 
+    &.floating {
+        position: fixed;
+        bottom: 40px;
+        right: 40px;
+        z-index: $z-index-400;
```

Comment:
Override both the default and pill shapes here, making the button round when it's icon only. All other scenarios may keep the button shapes 'as-is'. That way we don't have that weird oblong shape.

Reply:
It might help to have traditional button shape floating, I'll add a "round" shape in button.types